packages/neon-invoker/src/index.ts

```diff
@@ -9,12 +9,17 @@ export type RpcConfig = {
   networkMagic: number
 }
 
+export type TCalculateFee = {
```

Comment:
Can you drop the "T" of this type? We are not using this pattern

Reply:
fixed